components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -414,6 +428,12 @@ export class ReportingComponent implements OnInit, OnDestroy {
   getSuggestions(type: string, text: string, reportQuery: ReportQuery) {
     return this.suggestionsService.getSuggestions(type.replace('_id', ''), text, reportQuery).pipe(
       map(data => {
+
+        // Adding the option of 'no value' only to the control tag third level suggestions
+        if (type === 'control_tag_value') {
+          data.unshift(this.noValuesControlTagFilter);
+        }
+
```

Comment:
This is where I hardcoded our "no value" option, only when a user is filtering by Control Tags